lib/tasks/npm.rake

```diff
@@ -6,7 +6,7 @@ require 'English'
 namespace :npm do
   package_json = JSON.parse(File.read('package.json'))
 
-  package_json['scripts'].each do |name, _command|
+  package_json['scripts'].each_key do |name, _command|
```

Comment:
I think we can remove `_command` here, since `each_key` only grabs the keys (not the values)